libtelemetry/src/main/java/com/mapbox/services/android/telemetry/TelemetryClient.java

```diff
@@ -35,6 +35,10 @@ public TelemetryClient(String accessToken, String userAgent, TelemetryClientSett
     this.logger = logger;
   }
 
+  void setUserAgent(String userAgent) {
```

Comment:
get/set nomenclature is hijacked by the Java language and should be only used when defining POJOs/DTOs. What about `updateUserAgent` instead?